org.eclipse.mylyn-site/category.xml

```diff
@@ -128,6 +128,9 @@
    <feature id="org.eclipse.mylyn.github.feature" version="6.2.0.qualifier">
       <category name="github_connector"/>
    </feature>
+   <feature id="org.eclipse.egit.mylyn">
```

Comment:
It seems that the feature above has shadowed this one. Now it looks more consistent